### PR TITLE
User/orilevari/update debug readme

### PR DESCRIPTION
**Pull request by orilevari**
Rework custom operator README and include separate linked debug_readme document which explains in detail the intended usage of the debug operator. Also reworked directory structure to match user samples better.

### CODE REVIEW

**Comment by orilevari** [comment]
I also added a link in the base readme for the adapterselection sample since that was forgotten in its already approved and merged pull request

File: Samples/CustomOperatorCPU/debug_readme.md [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+2. Compare execution results on different hardware (CPU vs GPU).
+3. Visualize intermediate data for computer vision inference.
+
+## The Debug Operator
+Gaining access to intermediate data as it flows through your model during inference is achieved using a custom "Debug" operator. This operator is intended to be added into your model and consume a specific intermediate output as its input. It is further configurable with two additional attributes:
+1. file_type: [png|text] The file format to export intermediate outputs.
+
+	*png*: interprets and exports intermediate output as an image and requires tensor data in NCHW format. This is useful for computer vision scenarios where you seek to gain additional visual insight as to how features are being extracted through a model.
+
+	*text*: outputs raw tensor data in human readable format.
```
Comment (youngkim93) [inline Samples/CustomOperatorCPU/debug_readme.md]: is text space delimited?

File: Samples/CustomOperatorCPU/debug_readme.md [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+                              --debug_file_path DEBUG_FILE_PATH
+                              --intermediate_output INTERMEDIATE_OUTPUT
+    ```
+    You can also use [debug_all_outputs.py](customize_model/scripts/debug_all_outputs.py) to interlace Debug operators into every intermediate output of a graph.
+Here are netron visualizations of SqueezeNet's first section before and after running these scripts:
+
+
+    | | | |
+    |:-------------------------:|:-------------------------:|:-------------------------:|
+    |<img width="1604" src=customize_model/img/squeezenet.png>  Original | <img width="1604" src="customize_model/img/squeezenet_debug_single_output.png">[debug_single_output.py](customize_model/scripts/debug_single_output.py)|<img width="1604" src="customize_model/img/squeezenet_debug_all_outputs.png">[debug_all_outputs.py](customize_model/scripts/debug_all_outputs.py)|
```
Comment (youngkim93) [inline Samples/CustomOperatorCPU/debug_readme.md]: I think in the future we can just use the single python script to create all kinds of debug output models, but let's do that later in the future once Debug operator is officially supported on onnx runtime (winml)

File: Samples/CustomOperatorCPU/debug_readme.md [hunk 3]
```diff
@@ -0,0 +1,50 @@
...
+
+	*png*: interprets and exports intermediate output as an image and requires tensor data in NCHW format. This is useful for computer vision scenarios where you seek to gain additional visual insight as to how features are being extracted through a model.
+
+	*text*: outputs raw tensor data in human readable format.
+
+2. file_path: The file path to export intermediate outputs. Note that the parent directory of this file must exist. 
+		It is recommended for png Debug operators consuming tensor data with many channels to output to its own directory since a png will be created for each channel.
+
+## How to use the Debug Operator
+The Debug operator is planned be added to a future release of onnx runtime, but until then you will need to compile custom operator code into your app.
```
Comment (youngkim93) [inline Samples/CustomOperatorCPU/debug_readme.md]: as well as windows machine learning?

**Comment by orilevari** [comment]
This pull request also includes a small bug fix in the custom op sample app which was incorrectly converting relative path to absolute path.

**Pull request merged**